Header header
string first_name
string last_name
uint8 age
uint32 score
uint8 action
uint8 ACTION_FLOAT              = 0   # Do not Nothing
uint8 ACTION_MANUAL             = 1   # Control agv by manual